### ISSUE TITLE
Create Login Page for the App

### ISSUE THREAD

**Issue by Anushka-shukla** [neil-dev/YouSafe#8]
Do follow the wireframe once done.

**Comment by krish-bhanushali** [comment]
Hey, I would love to contribute in this great project.

**Comment by neil-dev** [comment]
Sure we would love to have you @krish-bhanushali 
However, we would be accepting PRs after October 1. So if you wish to submit a PR do it after that.

**Comment by krish-bhanushali** [comment]
@neil-dev Thank you, Can you assign this issue or it will be done after October 1

**Comment by neil-dev** [comment]
okay I'm assigning you this issue

**Comment by spacemonkey1101** [comment]
can I make the login page with kotlin

**Comment by neil-dev** [comment]
We would like to have the login page in Flutter. We will make a list of kotlin-related issues. You can check that out.

**Comment by neil-dev** [comment]
@krish-bhanushali are you working on the login page?

**Comment by krish-bhanushali** [comment]
Yes, @neil-dev I have almost completed my work and I did that according to the latest wireframe updated in the design but I am way behind the master branch so how should I proceed ? to contribute

**Comment by neil-dev** [comment]
@krish-bhanushali  You can delete the current fork, then fork the repo again. After that clone the fork, make the necessary changes and push them.

**Comment by krish-bhanushali** [comment]
@neil-dev done!
